src/NzbDrone.Core/Download/Clients/NzbVortex/NzbVortexSettings.cs

```diff
@@ -42,19 +43,22 @@ public NzbVortexSettings()
         [FieldDefinition(1, Label = "Port", Type = FieldType.Textbox)]
         public int Port { get; set; }
 
-        [FieldDefinition(2, Label = "Url Base", Type = FieldType.Textbox, Advanced = true, HelpText = "Adds a prefix to the NZBVortex url, e.g. http://[host]:[port]/[urlBase]/api")]
+        [FieldDefinition(2, Label = "Use SSL", Type = FieldType.Checkbox, HelpText = "Use secure connection when connecting to NZBVortex")]
```

Comment:
I think NZBVortex is the odd man out, where they only support SSL connections and using a reversing proxy to connect insecurely is indirectly. I think sticking with no setting makes sense.

Reply:
Updated :)